SpinTimer.cpp

```diff
@@ -46,7 +48,8 @@ SpinTimer::SpinTimer(SpinTimerAction* action, bool isRecurring, unsigned long ti
 {
   SpinTimerContext::instance()->attach(this);
 
-  if (0 < m_delayMillis)
+  if( (isAutostart == SpinTimer::IS_AUTOSTART) && 
+      (0 < m_delayMillis))
```

Comment:
This part of the condition shall be removed, because we want to depend just on `isAutostart` controlling the behaviour.
With `isAutostart = true` and `m_delayMillis = 0` we want to get the timer started right away (and let it expire as soon as possible, which is expressed by `m_delayMillis = 0`).